tests/tests_integration/test_api/test_events.py

```diff
@@ -64,7 +64,7 @@ def test_list(self, post_spy):
 
     def test_partitioned_list(self):
         # stop race conditions by cutting off max created time
-        maxtime = utils.timestamp_to_ms(datetime(2019, 6, 20, 17, 30))
+        maxtime = utils.timestamp_to_ms(datetime(2019, 12, 25, 17, 30))
```

Comment:
no idea why this was changed in this PR but it probably doesn't break anything

Reply:
this is unrelated :)